docs/README.md

```diff
@@ -285,8 +292,9 @@ For now CodeChecker supports only `C/C++` related analyzers:
 |                | [Cppcheck](/tools/report-converter/README.md#cppcheck)    |
 |                | [Clang Sanitizers](supported_code_analyzers.md#clang-sanitizers)    |
 
-We are planning to support multiple analyzers of different programming
-languages. For more detailed information [see](supported_code_analyzers.md).
+We are planning to support the storage of the results created by other
+analyzers for different languages (Java, Python ...). For more detailed
+information [see](supported_code_analyzers.md).
```

Comment:
This sentence after rendering the markdown to html is a bit interesting: "For more detailed information [see](supported_code_analyzers.md)."
What to see is missing from the sentence.
Maybe some thing like this?
"For more detailed information check out the [supported code analyzers](supported_code_analyzers.md) documentation."